core/common/src/main/java/alluxio/grpc/GrpcChannel.java

```diff
@@ -85,10 +85,26 @@ public void intercept(ClientInterceptor interceptor) {
 
   /**
    * Shuts down the channel.
```

Comment:
Shouldn't it be something like "Shuts down the communication channel." or "Shuts down the group channel."?

Reply:
Fixed.